src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import duke.exception.DukeException;
+import duke.task.TaskList;
+import duke.util.Parser;
+import duke.util.Storage;
+import duke.util.Ui;
+
+import java.io.File;
+
+
+public class Duke {
+    private final Parser PARSER;
+    private final Ui UI;
+
+    public static void main(String[] args) {
+        String filePath = "data" + File.separatorChar + "duke-storage.txt";
+        new Duke(filePath).run();
+    }
+
+    public Duke(String filePathToStorage) {
+        Storage DUKE_STORE = new Storage(filePathToStorage);
+        TaskList TASKLIST = TaskList.of(DUKE_STORE);
```

Comment:
Since `DUKE_STORE` and `TASKLIST` is not final, maybe you can consider camelCase instead? 
```suggestion
        Storage dukeStore = new Storage(filePathToStorage);
        TaskList taskList = TaskList.of(DUKE_STORE);
```